index.js

```diff
@@ -28,6 +28,10 @@ module.exports = {
   // prettier-plugin-css-order options
   cssDeclarationSorterOrder: 'smacss',
 
+  // prettier-plugin-sort-imports options
+  importOrder: ['^@', '^[a-zA-Z]', '^[./].*(?<!\\.s?css)$', '\\.s?css$'],
+  importOrderParserPlugins: ['decorators', 'jsx', 'typescript'],
```

Comment:
does jsx also cover tsx?

Reply:
Yes, that is handled with jsx and typescript.